templates/python-start-scrape-single-page/src/main.py

```diff
@@ -0,0 +1,24 @@
+from urllib.parse import urljoin
+
+import requests
+from apify import Actor
+from bs4 import BeautifulSoup
+
+async def main():
+    async with Actor:
+        # Read the Actor input
+        actor_input = await Actor.get_input() or {}
+        url = actor_input.get('start_url')
```

Comment:
Let's unify the naming - it should be just `url` - it is not just starting the crawling with this URL, it scrapes the data only from this one specific URL in this use case